README.md

```diff
@@ -87,17 +87,40 @@ $address->setCountryCode('US');
 $address->setPostalCode('10000');
 
 $xav = new \Ups\AddressValidation($accessKey, $userId, $password);
+
+$xav->validateReturnAVObject();
```

Comment:
Could you make sure that people understand this is optional and there used to be a different way of using this? It might cause inconvenience if people see something different than their original code, besides they might just want the easy version to use.

Also, I think it would be great if the examples also follow the coding guidelines, so including the right amount of white lines. In here out of nothing is one, but below in here all the if statements are glued together which would have a white line preferably.